mountpoint-s3/Cargo.toml

```diff
@@ -77,7 +77,6 @@ built = { version = "0.7.1", features = ["git2"] }
 [features]
 # Unreleased feature flags
 negative_cache = []
-sse_kms = []
```

Comment:
Perhaps an alternative approach in future is that we leave the feature flag in but remove all conditions throughout the code, then follow-up to remove it.